ueno/styles/ueno-black.json

```diff
@@ -0,0 +1,652 @@
+{
+	"settings": {
+		"border": {
+			"color": true,
+			"radius": true,
+			"style": true,
+			"width": true
+		},
+		"color": {
+			"custom": true,
+			"customGradient": true,
+			"link": true,
+			"palette": [
+				{
+					"color": "#bdbdbd",
+					"name": "Primary",
+					"slug": "primary"
+				},
+				{
+					"color": "#bdbdbd",
+					"name": "Secondary",
+					"slug": "secondary"
+				},
+				{
+					"color": "#bdbdbd",
+					"name": "Foreground",
+					"slug": "foreground"
+				},
+				{
+					"color": "#1b1818",
+					"name": "Background",
+					"slug": "background"
+				},
+				{
+					"color": "#1b1818",
+					"name": "Tertiary",
+					"slug": "tertiary"
+				}
+			],
+			"duotone": [
+				{
+					"slug": "foreground-to-background",
```

Comment:
I think should avoid the use of `foreground` and `background` as it doesn't make sense in a lot of contexts. What if we just called it `default` or `default-filter` like [Skatepark](https://github.com/Automattic/themes/blob/trunk/skatepark/theme.json#L38) does?